src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -441,7 +441,7 @@ public void GetIdElement()
             Assert.True(xel.InnerXml.StartsWith("<ObjectListTag"), "GetIdElement");
         }
 
-        [Fact]
+        [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
         public void GetPublicKey()
```

Comment:
What fails about this test?

That's another aspect of the general "these tests should be indicating what the behavior is with this OS configuration, instead of being turned off"

Reply:
Nothing. I had wrongly assumed any test that uses `signature` (which has `http://www.w3.org/2000/09/xmldsig#rsa-sha1`) would fail.